src/coverlet.core/Symbols/CecilSymbolHelper.cs

```diff
@@ -0,0 +1,290 @@
+//
+// This class is based heavily on the work of the OpenCover
+// team in OpenCover.Framework.Symbols.CecilSymbolManager
+//
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Coverlet.Core.Extensions;
+
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Mono.Collections.Generic;
+
+namespace Coverlet.Core.Symbols
+{
+    public static class CecilSymbolHelper
+    {
+        private const int StepOverLineCode = 0xFEEFEE;
+        private static readonly Regex IsMovenext = new Regex(@"\<[^\s>]+\>\w__\w(\w)?::MoveNext\(\)$", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
```

Comment:
Can you give an example of text that'll match this pattern

Reply:
Nit: Empty line after this

Reply:
https://github.com/OpenCover/opencover/issues/310